dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/config/DimensionDataCloudControlParserModule.java

```diff
@@ -17,18 +17,11 @@
 package org.jclouds.dimensiondata.cloudcontrol.config;
 
 import com.google.inject.AbstractModule;
-import com.google.inject.name.Names;
-import org.jclouds.Constants;
-import org.jclouds.json.config.GsonModule;
-
-import java.util.Collections;
 
 public class DimensionDataCloudControlParserModule extends AbstractModule {
 
    @Override
    protected void configure() {
-      Names.bindProperties(binder(), Collections.singletonMap(Constants.PROPERTY_PRETTY_PRINT_PAYLOADS, "true"));
-      bind(GsonModule.DateAdapter.class).to(GsonModule.Iso8601DateAdapter.class);
```

Comment:
We'll probably need to bind something like this adapter to be able to deserialize json strings to Date objects. Let's see which format the target API uses.